news/2023/2023-07-06-introducing-new-beatmap-pack-categories.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Introducing New Beatmap Pack Categories"
+date: 2023-07-06 09:00:00 +0000
+---
+
+Check out the latest and greatest new beatmap pack categories!
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/banner.png)
+
+I like maps. You like maps. We all like maps.
+
+For this reason, the mad scientists over at osu! labs have been synthesizing new ways to pump high-quality beatmaps directly into your bloodstream, and their latest creations are sure to excite you!
```

Comment:
I think having "the" reads better to me, just a suggestion though
```suggestion
For this reason, the mad scientists over at the osu! labs have been synthesizing new ways to pump high-quality beatmaps directly into your bloodstream, and their latest creations are sure to excite you!
```

Reply:
I feel that the completely absurd idea that osu! canonically has it's own established lab is funny enough to keep in